packages/space-opera/src/redux/initial_environment_images.ts

```diff
@@ -22,11 +22,11 @@ import {EnvironmentImage} from './lighting_state.js';
  */
 export const INITIAL_ENVIRONMENT_IMAGES: EnvironmentImage[] = [
   {
-    uri: '../shared-assets/environments/spruit_sunrise_1k_HDR.hdr',
+    uri: './node_modules/@google/model-viewer-shared-assets/environments/spruit_sunrise_1k_HDR.hdr',
     name: 'Spruit Sunrise'
   },
   {
-    uri: '../shared-assets/environments/aircraft_workshop_01_1k.hdr',
+    uri: './node_modules/@google/model-viewer-shared-assets/environments/aircraft_workshop_01_1k.hdr',
```

Comment:
Why this change, out of curiosity? Seems like it works both ways?

Reply:
It does, but this will work even if you serve from packages/space-opera directly, for the "npm run dev" case